lib/Backend/SymTable.h

```diff
@@ -9,21 +9,25 @@ class SymTable
     typedef JsUtil::Pair<SymID, Js::PropertyId> SymIdPropIdPair;
     typedef JsUtil::BaseDictionary<SymIdPropIdPair, PropertySym *, JitArenaAllocator, PrimeSizePolicy> PropertyMap;
     typedef JsUtil::BaseDictionary<Js::PropertyId, BVSparse<JitArenaAllocator> *, JitArenaAllocator, PowerOf2SizePolicy> PropertyEquivBvMap;
-public:
-    static const int    k_symTableSize = 8192;
-    Sym *               m_table[k_symTableSize];
-    PropertyMap *       m_propertyMap;
-    PropertyEquivBvMap *m_propertyEquivBvMap;
 
 private:
+	static const int    k_symTableSize = 8192;
+	static const int    k_maxImplicitParamSlot = 4;
+    Sym *               m_table[k_symTableSize];
+	StackSym *          m_implicitParams[k_maxImplicitParamSlot];
+    PropertyMap *       m_propertyMap;
     Func *              m_func;
     SymID               m_currentID;
     SymID               m_IDAdjustment;
 
 public:
+	PropertyEquivBvMap *m_propertyEquivBvMap;
+
+public:
     SymTable() : m_currentID(0), m_func(nullptr), m_IDAdjustment(0)
```

Comment:
just noticed that this is initialized in Init(). So this is a nit.

---
In reply to: [75063468](https://github.com/Microsoft/ChakraCore/pull/1373#discussion_r75063468) [](ancestors = 75063468)